docs/maintain/maintain-wss.md

```diff
@@ -1,86 +1,112 @@
 ---
 id: maintain-wss
-title: Set up Secure WebSocket for Remote Connections
-sidebar_label: Set up Secure WebSocket for Remote Connections
+title: Secure WebSocket
+sidebar_label: Secure WebSocket
 description: Steps on setting up a secure socket for remote connections.
 keywords: [web socket, remote, connection, secure websocket]
 slug: ../maintain-wss
 ---
 
-You might want to host a node on one server and then connect to it from a UI hosted on another, e.g.
-[Polkadot-JS UI](https://polkadot.js.org/apps). This will not be possible unless you set up a secure
-proxy for websocket connections. Let's see how we can set up WSS on a remote Substrate node.
+A websocket provides access to the RPC server part of a substrate node, which can be used as a way
+to access the underlying network and/or validator node. By default you can access your node's RPC
+server from localhost (for example to rotate keys or do other maintenance) but to access it from
+from another server or for example connect to it from on UI like
+[Polkadot-JS UI](https://polkadot.js.org/apps) you will need to enable a secure proxy for the
+websocket connection.
 
-_Note: this should **only** be done for sync nodes used as back-end for some dapps or projects.
-Never open websockets to your validator node - there's no reason to do that and it can only lead to
-security gaffes._
-
-In this guide we'll be using Ubuntu 18.04 hosted on a \$10 DigitalOcean droplet. We'll assume you're
-using a similar OS, and that you have nginx installed (if not, run `sudo apt-get install nginx`).
+:::note Enabling remote access to your validator node should not be neccessary and can only lead to
+security problems :::
```

Comment:
```suggestion
:::note Enabling remote access to your validator node should not be necessary and is not
suggested as it can often lead to security problems :::
```